### DISCUSSION TITLE
PhysX 4 tracked issues

### DISCUSSION THREAD

**Discussion by shaneasd** [NVIDIA-Omniverse/PhysX#221] [Q&A] [answered]
Are the Issues in the PhysX 4 repository still tracked by the PhysX 5 development team? I ask because it appears https://github.com/NVIDIAGameWorks/PhysX/issues/436 is not yet fixed in PhysX 4 _or_ 5, and I'm wondering if it would be helpful or unhelpful for me to reraise it in this repository.

**Comment by preist-nvidia** [comment, marked as answer]
Hi @shaneasd - yes we are tracking both repositories' issues and discussions. 436 slipped through our cracks though, but we'll have a look, thanks for reraising it.
